realm/realm-library/src/main/cpp/jni_util/exception_utils.cpp

```diff
@@ -0,0 +1,65 @@
+#include <unwind.h>
```

Comment:
Did you see anything about the API 16 with libunwind? I cannot find too much information about that.

But! https://code.google.com/p/android/issues/detail?id=199936
> libunwind is not an NDK API. even if you get this to work now, in future releases you will be prevented from linking against libunwind at runtime.

:( :( :(